docs/reference/redirects.asciidoc

```diff
@@ -158,10 +158,21 @@ See <<grok,Grokking grok>>.
 
 // [START] Security redirects
 
+[role="exclude",id="security-minimal-setup"]
```

Comment:
Yes! This inclusion is a redirect for the removed page so that if users arrive at that URL (such as from a Google search), they have a means to find the page that they're looking for.